### PR TITLE
Local guilty commit search

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**
Features:
1. Guilty commit search on local machine
2. Store PyTorch dependencies including vision, audio, text and data in Docker image which is used for guilty commit search. Docker image size increased from 12.6G to 15.1G.
3. Inductor benchmark pipeline locally: trigger target and baseline inductor benchmark test, and compare the report in one pipeline

### CODE REVIEW

**Review by chuanqi129 — APPROVED** [review]
Overall LGTM, it seems need to be rebased after the PR #97 merged, right?

**Comment by WeizhuoZhang-intel** [comment]
> Overall LGTM, it seems need to be rebased after the PR #97 merged, right?

Yes, it's based on PR #97

**Pull request merged**
